### PR TITLE
make `shell-cmd` function more concise

### PR DESCRIPTION
**Pull request by anmonteiro**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -25,24 +25,10 @@ Add the following to your `~/.emacs` or `~/.emacs.d/init.el` file:
...
-  (let ((stdoutput (chomp-end
-                    (with-output-to-string
-                      (with-current-buffer
-                          standard-output
-                        (process-file shell-file-name nil
-                                      '(t nil)  nil
-                                      shell-command-switch cmd))))))
-    (when (not (= (length stdoutput) 0))
-      stdoutput)))
+  (car (ignore-errors (apply 'process-lines (split-string cmd)))))
```
Comment (arichiardi) [inline README.md:31]: Way better, but maybe we don't want to ignore-errors?

Comment (anmonteiro) [reply README.md:31]: It's exactly what the previous version did though. We wanna return `nil`

Comment (arichiardi) [reply README.md:31]: Saw that below yes

**Review by arichiardi — APPROVED** [review]
Thanks!

**Pull request merged**